packages/contracts/test/utils/coverage.ts

```diff
@@ -0,0 +1,21 @@
+import { devConstants } from '@0xproject/dev-utils';
+import { CoverageSubprovider, ZeroExArtifactAdapter } from '@0xproject/sol-cov';
+import * as fs from 'fs';
+import * as _ from 'lodash';
+
+let coverageSubprovider: CoverageSubprovider;
+
+export const coverage = {
+    getCoverageSubproviderSingleton(): CoverageSubprovider {
+        if (_.isUndefined(coverageSubprovider)) {
+            coverageSubprovider = coverage._getCoverageSubprovider();
+        }
+        return coverageSubprovider;
+    },
+    _getCoverageSubprovider(): CoverageSubprovider {
+        const defaultFromAddress = devConstants.TESTRPC_FIRST_ADDRESS;
+        const config = JSON.parse(fs.readFileSync('compiler.json').toString());
```

Comment:
`compiler.json` should be a constant.

Reply:
What if the file doesn't exist? Handle that.

Reply:
How should I improve the standard error message that will say smth like: `compiler.json` doesn't exist?